targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile

```diff
@@ -200,6 +200,16 @@ RUN APT_INSTALL="apt-get install -y --no-install-recommends" && \
 # Install openssh
   $APT_INSTALL openssh-server
 
+# ==================================================================
+# Apex for PyTorch mixed precision training
+# ==================================================================
+
+RUN mkdir /apex && \
+    cd /apex && \
+    git clone https://github.com/NVIDIA/apex && \
+    cd apex && \
+    python -m pip install -v --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./
```

Comment:
1. Let's add tests
2. let's install it to the right place

Reply:
1) We can use tests that are provided by py library developers (see: apex/tests/L0). 
2) What is the proper place Apex must be installed to?

Reply:
fixed both